src/test/ui/associated-types/impl-trait-return-missing-constraint.stderr

```diff
@@ -1,14 +1,16 @@
 error[E0271]: type mismatch resolving `<impl Bar as Foo>::Item == i32`
-  --> $DIR/impl-trait-return-missing-constraint.rs:25:13
+  --> $DIR/impl-trait-return-missing-constraint.rs:26:5
    |
 LL | fn bar() -> impl Bar {
-   |             -------- the found opaque type
+   |             -------- the expected opaque type
 ...
-LL | fn baz() -> impl Bar<Item = i32> {
```

Comment:
*additionally* point at the `Item = i32`

*additionally* point at `trait Bar`'s associated type